aicsimageio/metadata/README.md

```diff
@@ -0,0 +1,21 @@
+# AICSImage Metadata Module
+
+Undecided how to organize this module. This likely isn't included in 4.0 but more
+likely 4.1 or similar but good to start thinking about.
+
+I am currently thinking:
+```
+aicsimageio/
+    metadata/
+        omexml.py  # our remade and updated omexml object generated from source XSD
+        transform.py  # the transform functions to convert to ome
+        xslt/
+            czi.xslt  # from czi to ome
+            3i.xslt  # from 3i to ome
```

Comment:
it occurs to me that maybe the format-specific files like czi.xslt should live closer to where CZIReader lives?  Maybe...?

Reply:
I can see an argument for that as well. Since I _expect_ this to be a 4.1 change we can open it up for discussion closer to then? @heeler @AetherUnbound thoughts?

Reply:
I had forgotten that we were going to keep that stuff in a git submodule. (I'm still not sure that's best) In that case, I guess my thought is to associate the conversion to ome with the reader by having a Reader.get_ome_metadata or some such method that does the conversion.  Encapsulating file format specific code within only one Reader subclass.

Reply:
Yea I touch a bit upon this in the newly written Roadmap document -- again I think this can somewhat be decided in 4.1 but I am starting to side towards each reader having their own submoduled metadata converter